rke/k8s_rke_system_images.go

```diff
@@ -6614,6 +6614,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.6.5
```

Comment:
The current plan is to release June patches with v2.6.7, so let's update comment for this and newly added entries to:
```suggestion
		// Enabled with v2.6.7
```

The issues are still tracked in their own milestone, but likely will be updated to v2.6.7 too.